mizar/daemon/interface_service.py

```diff
@@ -165,6 +166,8 @@ def _ProvisionVethInterface(self, interface, cni_params):
         # Configure the Transit Agent
         self._ConfigureTransitAgent(interface)
 
+        self._UpdateSubstrateInterface(interface)
```

Comment:
We don't need this, it can be removed.
The transit xdp program does not need to update/delete the substrate information about its own node.